### PR TITLE
Feature/parking theme

### LINKED ISSUE #1331: Missing styling of the parking portal
Located in html/parking

It needs styling while staying lightweight

### PR DESCRIPTION
**Pull request by cgx**
# Description

Design the HTML templates of the parking portal to reflect the changes made to the captive portal.
# Impacts

CSS of captive portal has been moved to the `html/common` directory so it can be shared with the parking portal.
# Issue

Fixes #1331
# Delete branch after merge

YES

### CODE REVIEW

**Comment by julsemaan** [comment]
The parking portal doesn't have access to the common directory.

Will fix in devel and merge this after if it works

**Comment by cgx** [comment]
All right!

File: html/parking/back-on-network.html [hunk 1]
```diff
@@ -1,4 +1,52 @@
...
+          <div class="box box--hero u-m u-mb--">
+            <img class="iconic" src="/common/img/ic_done_24px.svg" alt="Info"
+                 onerror="this.onerror=null; this.src='/common/img/ic_done_72px.png'">
+          </div>
+          <h4>Restrictions have been removed</h4>
+        </div>
 
+        <div class="layout layout--center">
+          <div class="layout__item u-2/3 u-1/1-palm">
+            <a href="/cgi-bin/release.pl" onclick="release('btn-label')" class="btn btn--full btn--light u-mt">
```
Comment (julsemaan) [inline html/parking/back-on-network.html]: the href here should point to an external website so the redirection process brings the user back to the normal portal

File: html/parking/index.html [hunk 2]
```diff
@@ -1,10 +1,53 @@
...
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="/common/styles.css" media="screen" rel="stylesheet" type="text/css" />
+    <title>Parking</title>
+    <script>
+      function release(target){
+        document.getElementById(target).innerHTML = "One moment please";
```
Comment (julsemaan) [inline html/parking/index.html]: The click event of the button needs to be disabled also to prevent bash clicking.

**Comment by julsemaan** [comment]
Other than the two fixes I pointed, this is good to merge

**Comment by cgx** [comment]
Should be good to merge.

**Pull request merged**